docs/source/actionchain.rst

```diff
@@ -87,3 +87,64 @@ Note:
 
 * Output of an action elements is always prefixed by element name. e.g. In ``{"cmd":"echo c2 {{c1.localhost.stdout}}"}`` `c1.localhost.stdout` refers to the output of 'c1' and further drills down into properties of the output.
 * A special ``__results`` key provides access to the entire result upto that point of execution.
+
+Error Reporting
+~~~~~~~~~~~~~~~
+
+ActionChain errors are classified as:
+
+* Errors reported by a specific task in the chain. In this case the error is reported as per behavior of the particular action in the task.
+
+Sample -
+
+::
+
+   "result": {
+        "tasks": [
+            {
+                "created_at": "2015-02-27T19:29:02.057885+00:00",
+                "execution_id": "54f0c57e0640fd177f278052",
+                "id": "c1",
+                "name": "c1",
+                "result": {
```

Comment:
There could also be a case when `result` attribute contains an `error` key (e.g. running an action for some reason fails).